README.md

```diff
@@ -1,3 +1,33 @@
 # Go client and cli for [outline][def]
 
-[def]: https://www.getoutline.com/
\ No newline at end of file
+[def]: https://www.getoutline.com/
+
+
+
+# Usage
+
+
```

Comment:
```suggestion
```

Reply:
ahh! I ususally keep more than one empty line between headings so as to differentiate with paragraphs of one section which are also separated by one line. looks a bit more clean in text-view. But anyways I will get rid of them.